modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/util/localizable/transform.es.js

```diff
@@ -25,6 +25,17 @@ export function convertValueToJSON(value) {
 	return value;
 }
 
+export function convertStringToObject(value, localeId) {
+	if (typeof value === 'string') {
+		const object = new Object();
+		object[localeId] = value;
```

Comment:
@pedro-oliveira446 Please, add a blank line between the const definiton and its usage. Thanks :)